topics/admin/tutorials/ansible-galaxy/tutorial.md

```diff
@@ -646,220 +642,150 @@ NGINX plugins are dynamic shared objects since [1.9.11](https://www.nginx.com/bl
 >
 > 1. Edit your `group_vars/galaxyservers.yml`, we will update the line that `http: 0.0.0.0:8080` to be `socket: 127.0.0.1:8080`. This will cause uWSGI to only respond to uWSGI protocol, and only to requests originating on localhost.
 >
-> 2. Add the role `geerlingguy.nginx` to your playbook and have it run as root.
+> 2. Add the role `galaxyproject.nginx` to your playbook and have it run as root.
 >
-> 3. In our group variables file, we will set a convenience variable, `hostname`, to refer to our VM. If you have a DNS entry for your server, add `hostname: <your server name>` to your group variables. Otherwise, add {% raw %}`hostname: "{{ ansible_hostname }}"`{% endraw %}.
->
-> 4. We need to configure the virtualhost. This is a slightly more complex process as we have to write the proxying configuration ourselves. This may seem annoying, but it is often the case that sites have individual needs to cater to, and it is difficult to provide a truly generic webserver configuration.
+> 3. We need to configure the virtualhost. This is a slightly more complex process as we have to write the proxying configuration ourselves. This may seem annoying, but it is often the case that sites have individual needs to cater to, and it is difficult to provide a truly generic webserver configuration. Additionally because it's 2019, we will include certbot for SSL from the start. There is no longer any valid excuse for not deploying SSL.
```

Comment:
```suggestion
> 3. We need to configure the virtualhost. This is a slightly more complex process as we have to write the proxying configuration ourselves. This may seem annoying, but it is often the case that sites have individual needs to cater to, and it is difficult to provide a truly generic webserver configuration. Additionally, we will enable secure communication via HTTPS using SSL/TLS certificates provided by [certbot](https://certbot.eff.org/).
```